CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+#### Contributing
+
+When contributing to this repo, also increment the version number in `nougat/__init__.py` as part of your pull request. If you suspect other pull requests will be merged before yours, simply ask the responsible reviewer to suggest a number after their review. Versioning follows the typical `MAJOR.MINOR.REVISION` system. Version numbers are set as follows:
+
+- Revision - Small sized updates such as addressing a logic error or changing what version of a dependency is required
+- Minor - Medium sized updates such as adding a new functionality or optimizing an existing feature
+- Major - Large sized updates that either add a big bundle of functionality, or optimization that required a large section of the repository to be refactored.
```

Comment:
Awesome! However, I'll probably apply a very liberal interpretation of this schema.